### PR TITLE
fix: re-deploy Che via OLM installer.

### PR DESCRIPTION
**Pull request by AndrienkoAleksandr**
Signed-off-by: Oleksandr Andriienko <oandriie@redhat.com>

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/CRW-2363

### How to test this PR?
1. Install Che using OLM installer:

```bash
./run server:deploy --installer olm -p openshift --olm-channel=tech-preview-stable-all-namespaces
```

2. Uninstall Che:

```bash
./run server:delete -n eclipse-che -y
```

3. Install one more time using 1 step again. Re-installation should be successfull.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: src/tasks/component-installers/devfile-workspace-operator-installer.ts [hunk 1]
```diff
@@ -104,11 +104,25 @@ export class DevWorkspaceTasks {
...
    * Returns list of tasks which uninstall dev-workspace.
    */
   getUninstallTasks(): ReadonlyArray<Listr.ListrTask> {
     return [
+      {
+        title: 'Check DevWorkspace OLM installation',
+        task: async (ctx: any, task: any) => {
+          ctx.isDevWorkspaceInstalledViaOLM = Boolean(await this.isDevWorkspaceInstalledViaOLM())
+
+          task.title = await `${task.title} ...OK ${ctx.isDevWorkspaceInstalledViaOLM}`
```
Comment (tolusha) [inline src/tasks/component-installers/devfile-workspace-operator-installer.ts:123]: ...[Installed/Not installed] ?

**Review by mmorhun — APPROVED** [review]

**Review by tolusha — APPROVED** [review]
We are good to merge

**Pull request merged**